HTML/html_completions.py

```diff
@@ -396,6 +396,14 @@ def get_tag_attributes():
 
     return tag_attr_dict
 
+def get_boolean_attributes():
```

Comment:
As those boolean attributes are not needed anywhere else, you could probably directly return a boolean result about whether a given attribute is a boolean or not.

Using `set`s is also slightly faster.

```suggestion
def is_bool(attr):
    return attr in {
        'async', 'autofocus', 'autoplay', 'checked', 'contenteditable',
        'controls', 'default', 'defer', 'disabled', 'formNoValidate',
        'frameborder', 'hidden', 'ismap', 'itemscope', 'loop', 'multiple',
        'muted', 'nomodule', 'novalidate', 'open', 'readonly', 'required',
        'reversed', 'scoped', 'selected', 'typemustmatch'
    }
```